src/ansys/templates/python/solution/{{cookiecutter.__project_name_slug}}/README.rst

```diff
@@ -1,15 +1,24 @@
-#######################################
-Ansys Solutions {{cookiecutter.solution_name}}
-#######################################
+#################################################
+Ansys Solutions {{cookiecutter.solution_name}} 🚀
+#################################################
```

Comment:
I am a big fan of icons but I think they are discouraged in technical documentation. Hence, I would suggest to remove it:

```suggestion
##############################################
Ansys Solutions {{cookiecutter.solution_name}}
##############################################
```

Reply:
@jorgepiloto I removed all icons.